guide/english/php/forms/handling-form-input/index.md

```diff
@@ -6,7 +6,8 @@ GET VS POST
 
```

Comment:
Here the Heading seems to dictate `Get vs POST` which seems odd when you add the request variable as well. It would suit this article well if you instead change this heading to something that suggests the use of all 3 variables to receive form-inputs